fastapi.Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM --platform=linux/amd64 python:3.8-slim
+
```

Comment:
Maybe move this file into the deployment tutorial folder. It make more sense there